change/@fluentui-web-components-ec172ca8-23f3-4d01-a969-6504fa67a87a.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33023&commitId=aa88013961b5ae2dc58dd97bb3d2a1d01c3046d1&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.Badge Mask RTL.chromium.png|3| Changed|
|Avatar Converged.badgeMask.chromium.png|12| Changed|

</details>


  <details><summary><b>AvatarGroup Converged</b> 12 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|AvatarGroup Converged.Layout Pie Dark Mode.chromium.png|4| Changed|
|AvatarGroup Converged.Layout Pie 1 RTL.chromium.png|6| Changed|
|AvatarGroup Converged.Layout Pie 2 High Contrast.chromium.png|50| Changed|
|AvatarGroup Converged.Layout Pie 2 RTL.chromium.png|6| Changed|
|AvatarGroup Converged.Layout Pie 1 Dark Mode.chromium.png|4| Changed|
|AvatarGroup Converged.Layout Pie 1 High Contrast.chromium.png|50| Changed|
|AvatarGroup Converged.Layout Pie 2 Dark Mode.chromium.png|4| Changed|
|AvatarGroup Converged.Layout Pie 2.chromium.png|6| Changed|
|AvatarGroup Converged.Layout Pie RTL.chromium.png|6| Changed|
|AvatarGroup Converged.Layout Pie High Contrast.chromium.png|50| Changed|
|AvatarGroup Converged.layoutPie.chromium.png|6| Changed|
|AvatarGroup Converged.layoutPie-2.chromium.png|6| Changed|

</details>


  <details><summary><b>Drawer</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.overlay drawer full.chromium.png|3281| Changed|

</details>


  <details><summary><b>TagPicker</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|TagPicker.Expand Icon Dark Mode.chromium.png|27| Changed|
|TagPicker.Secondary Action Dark Mode.chromium.png|27| Changed|
|TagPicker.Expand Icon High Contrast.chromium.png|145| Changed|
|TagPicker.Secondary Action High Contrast.chromium.png|145| Changed|

</details>


  <details><summary><b>Tree</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Tree.Actions Dark Mode.chromium.png|3589| Changed|
|Tree.Actions High Contrast.chromium.png|41502| Changed|
|Tree.Actions RTL.chromium.png|35| Changed|
|Tree.actions.chromium.png|36| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>